### PR TITLE
DIV-5457-Add downloadable files to session

### PR DESCRIPTION
**Pull request by jacmurphy**
# Description
https://tools.hmcts.net/jira/browse/DIV-5457
Linked to : https://github.com/hmcts/div-decree-nisi-frontend/pull/527/files
Add session data for documents

### CODE REVIEW

File: app/steps/issued-to-bailiff/index.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
   get url() {
     return '/issued-to-bailiff';
   }
+
+  interceptor(ctx, session) {
+    session.downloadableFiles = this.getDownloadableFiles(session);
+    return ctx;
+  }
+
+  getDownloadableFiles(session) {
```
Comment (madjava) [inline app/steps/issued-to-bailiff/index.js:14]: @jacmurphy Any unit test for these? Something basic would do as well.

Comment (jacmurphy) [reply app/steps/issued-to-bailiff/index.js:14]: Tests added. Thanks for pointing it out!

File: test/fixtures/mockIssuedToBailiffSession.json [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  ],
+  "screenHasMarriageCert": "Yes",
+  "respondentFirstName": "PetitonerN",
+  "reasonForDivorceLimitReasons": "false",
+  "reasonForDivorceHasMarriageDate": "true",
+  "marriedInUk": "Yes",
+  "marriageDateMonth": 1,
+  "createdDate": "2020-07-08T00:00:00.000+0000",
+  "marriageCanDivorce": "true"
+}
```
Comment (madjava) [inline test/fixtures/mockIssuedToBailiffSession.json:259]: Nice. I think for your test you can remove more of these content, only the d8 part should cut it. If you fancy then do 🙂

**Comment by madjava** [comment]
Nice one 👌🏾

**Review by madjava — APPROVED** [review]

**Review by Dizrupto — APPROVED** [review]

**Pull request merged**